config/scripts/relay-replay-buffer-benchmark.mjs

```
@@ -0,0 +1,212 @@
+  new URL('../../src/relay/pty-handler.ts', import.meta.url),
+  'utf8'
+)
+if (!/managed\.buffered\.append\(/.test(HANDLER_SOURCE)) {
+  throw new Error('relay no longer appends into a chunk deque; this benchmark is stale')
+}
+const capMatch = HANDLER_SOURCE.match(/REPLAY_BUFFER_MAX = ([\d *]+)/)
+if (!capMatch) {
+  throw new Error('REPLAY_BUFFER_MAX not found; this benchmark is stale')
+}
+// eslint-disable-next-line no-eval -- reading a numeric literal expression out of source
+const REPLAY_BUFFER_MAX = Number(eval(capMatch[1]))
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)